packages/react/src/ui/Connect/index.tsx

```diff
@@ -17,13 +17,21 @@ import {
 import { getThemeVariables } from './themes';
 
 export function Connect() {
+  // Fix hydration problem between nextjs render and frontend render
+  // UI was not getting updated and theme colors was set wrongly
+  // see more here https://nextjs.org/docs/messages/react-hydration-error
+  const [isClient, setIsClient] = useState(false);
```

Comment:
I think for this case we could simply add a `suppressHydrationWarning` to the `FuelRoot`, to prevent this extra re-render.

```tsx
<FuelRoot suppressHydrationWarning ... />
```

What you think?

Reply:
the problem is not the warning, but the UI not being updated with right colors
I will try this approach

Reply:
yeah as I expected the warning doesnt show anymore but the UI still broken with light theme


<img width="618" alt="image" src="https://github.com/FuelLabs/fuels-npm-packs/assets/8636507/a7327e46-d114-46e4-9b03-9a74951232f8">